scripts/install-ohmyzsh.sh

```diff
@@ -2,7 +2,13 @@
 
 # Install oh-my-zsh
 
-git clone git://github.com/robbyrussell/oh-my-zsh.git /home/vagrant/.oh-my-zsh
+if [ -f /home/vagrant/.oh-my-zsh ]
+then
+    echo "oh my zsh already installed."
+    exit 0
+fi
+
```

Comment:
Need to add a line here that touches the `/home/vagrant/.oh-my-zsh` like we do in other cases.

Reply:
Oh wait, no, we need to NOT rely on the `.oh-my-zsh` folder existing and instead touch another file. This is the paradigm we're moving towards for v8

Reply:
Ok cool. You mean like this for example right?

https://github.com/laravel/homestead/blob/master/scripts/install-mysql8.sh#L12

Reply:
Exactly!